docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md

```diff
@@ -0,0 +1,228 @@
+---
+id: router-configuration
+title: Router Configuration
+sidebar_label: Configuration
+hide_table_of_contents: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This article expands on the [Linux and Docker router deployment](/guides/deployments/10-linux/20-router/10-deploy.mdx) article with configuration concepts and examples. [Kubernetes is covered separately](/guides/deployments/30-kubernetes//kubernetes-router.mdx).
+
+## Config Management
+
+You may create or edit the configuration file manually or run `ziti create config router` to generate one. The generator aims to accelerate developing a working configuration with the most relevant options. The output can be influenced with arguments and variables. Run `ziti create config environment` for descriptions of the available environment variables.
```

Comment:
In general, I'd like to avoid phrases like "you may". I'd like to favor more direct phrasing. "To create an initial configuration file, run....". This is also reference-style doc. It should be more direct and to the point. It might not even require the "To" preamble and be more direct even. "do this thing by running this command"